lib/xform.js

```diff
@@ -34,6 +47,7 @@ var XFormState = State.extend(function (self, name, opts) {
         self.states.add('states:example-xformState', function(name){
             return new XFormState(name, {
                 next: "states:end",
```

Comment:
Maybe we should show a function for `next` since that's probably the most common usage and we can show the `f(xform)`.